cli/CHANGELOG.md

```diff
@@ -1,4 +1,12 @@
 <!-- See the ../guides/writing-the-cypress-changelog.md for details on writing the changelog. -->
+# 13.2.0
```

Comment:
I figured a minor version bump because it is a larger change of dependencies which could have broader reach than a patch. But semver wise I'm OK with either

I will update the title in the PR description

Reply:
Technically this is a chore, as opposed to a fix or feature, so it really shouldn't trigger a release, since there's nothing user-facing. But I'm not sure our changelog validation supports adding the dependency update line without it being tied to a release, so I think patch version is the way to go since this doesn't add any functionality for users.

Reply:
true and that would be logically consistent with how we handled the webpack update. I also need to sync the branch and see if any features have made their way into `develop`